client/src/components/Graph/index.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div>
+    <md-toolbar class="md-large">
+      <div class="md-toolbar-row">
+        <div class="md-toolbar-section-start">
+          <h3 class="md-title">Plot</h3>
+        </div>
+      </div>
+      <md-field v-if="plotChoices">
+        <label for="plotChoices">Plot Type</label>
+        <md-select v-model="plotChoice" id="plotChoices" name="plotChoice">
+          <md-option
+            v-for="choice in plotChoices"
+            :key="choice"
+            :value="choice"
+          >
+            {{choice}}
+
+          </md-option>
+        </md-select>
+      </md-field>
+      <Plot
+        :id="plotElementId"
+        v-if="plotDataExists()"
+        :plotData="READ_ONLY.plotData"
+        :plotType="READ_ONLY.plotType"
+        :plotElementId="plotElementId"
+      />
+      <p id="no-graph-notification" v-else>No graph has been generated yet</p>
+      <Error v-if="READ_ONLY.error" :type="READ_ONLY.errorType" :message="READ_ONLY.errorMessage"/>
+    </md-toolbar>
+  </div>
+</template>
+
+<script>
+import Plot from './Plot.vue';
+import Error from '../Error.vue';
+
+export default {
+  name: 'Graph',
+  data() {
+    return {
+      READ_ONLY: this.$store.state,
+      plotChoices: null,
+      plotChoice: null,
+      plotElementId: 'd3-chart',
+    };
+  },
+  components: {
+    Plot,
+    Error,
+  },
+  async created() {
+    const plotChoices = [...this.READ_ONLY.plotTypes];
+    if (plotChoices.includes('dndm')) {
+      this.plotChoice = 'dndm';
```

Comment:
That is actually taken out of the most recent code that will is being created for the plot display. It was a preventative measure, in case the name of the plot changed, it would still choose the first plot that is listed.

Reply:
OK, cool.